ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master.yaml

```diff
@@ -0,0 +1,27 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: '4.0'
+  namespace: ocp
+base_images:
+  base:
+    name: '4.0'
+    namespace: ocp
+    tag: base
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 400Mi
+build_root:
+  project_image:
+    dockerfile_path: Dockerfile
+    context_dir: ci.Dockerfile
```

Comment:
This looks like the cause of the rehearsal errors - looking at https://github.com/operator-framework/community-operators it looks like `ci.Dockerfile` is the actual Dockerfile, not a context directory.

```
build_root:
  project_image:
    dockerfile_path: ci.Dockerfile
```